packages/hardhat/contracts/strategy/CrabStrategyV2.sol

```diff
@@ -28,6 +28,36 @@ import {Power2Base} from "../libs/Power2Base.sol";
 import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
 import {ECDSA} from "@openzeppelin/contracts/cryptography/ECDSA.sol";
 
+/**
+ * Crab V2 Error Codes:
+ * C1: Caller is not timelock
+ * C2: Contract not yet initialized
+ * C3: Invalid oracle address
+ * C4: Invalid timelock address
+ * C5: Invalid ETH:WSqueeth address
+ * C6: Invalid crabMigration address
+ * C7: Invalid hedge time threshold
+ * C8: Invalid hedge price threshold
+ * C9: Cannot receive ETH
+ * C10: Caller not Crab Migration contract
+ * C11: Crab V2 already initialized
+ * C12: Squeeth contracts not shut down
+ * C13: Crab must redeemShortShutdown
+ * C14: Twap period is too short
+ * C15: Price tolerance is too high
+ * C16: Deposit exceeds strategy cap
+ * C17: Clearing Price should be below bid price
+ * C18: Clearing Price should be above offer price
+ * C19: Invalid offer signature
+ * C20: Order has expired
+ * C21: Manager Price should be greater than 0
+ * C22: Not a valid Time or Price hedge
```

Comment:
Do we want to be more precise with this? Should be fine, but Sherlock said: "Time or Price is not within range" was too vague, so this could be considered the same.